modules/preview-react/_examples/stories/SelectWithFormik.stories.mdx

```diff
@@ -0,0 +1,10 @@
+import {Meta, Story} from '@storybook/addon-docs';
+import {SelectWithFormik} from './examples/SelectWithFormik';
+
+<Meta title="Examples/Select With Formik/React" />
```

Comment:
Thoughts on changing the Meta title to `Examples/Select/React` to be more consistent with our CookieBanner and Header examples which only show the component name at the top level?

We could also do `Examples/Inputs/Select/React` (and `Examples/Inputs/Text Input/React`).

Reply:
Good call. I think I'll keep it consistent with CookieBanner and Header. Maybe when there are more examples additional nesting would be helpful.